src/components/pages/JudgeTable/JudgeTable.js

```diff
@@ -305,11 +307,12 @@ export default function JudgeTable(props) {
             tab={
               <Button
                 className="save-cases-btn"
+                disabled={isDisabled}
```

Comment:
Smart - I love the way you use state to determine if this button should even be pressed. Subtle way to tell the user some action is required on their part before this option becomes available.